pages/_app.tsx

```diff
@@ -34,18 +32,9 @@ const CustomApp = ({
   pageProps,
 }: ExtendedAppProps): JSX.Element | null => {
   const [user] = useState(pageProps.user);
-  const features: FeatureSet = {
-    // FEATURE-FLAG-EXPIRES [3000-12-31]: feature-flags-implementation-proof
-    'feature-flags-implementation-proof': {
-      isActive: pageProps.environmentName === 'development',
-    },
-
-    /*
-    The feature-flags-implementation-proof has been setup to have an expiry date in the far future.
-    The FEATURE-FLAG-EXPIRES comment above will cause ESLint errors once the date in the square brackets has passed.
-    Add feature flags below following the format in the example shown.
-    */
-  };
+  const features = getFeatureFlags();
+  features['feature-flags-implementation-proof'].isActive =
+    pageProps.environmentName === 'development';
```

Comment:
Couldn't figure out a way to reference `pageProps` in the new `features.ts` file. 
Is there a way to do this so that we don't have to set the condition outside of the new file?
Open to any suggestions/feedback 😅

Reply:
I'd pass an object into `getFeatureFlags()` which has any data we need to determine if a flag is active. Something like:

```ts
const features = getFeatureFlags({
  environmentName: pageProps.environmentName
});
```

We can extend that object in the future with anything else that might be useful (e.g. the current route)

Reply:
Done in 4735a4f